Group3-Backend-Spring/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,11 @@
+server.port=8080
+
+jwt.secret = javainuse
+jwt.expirationDateInMs=180000
+jwt.refreshExpirationDateInMs=9000000
+
+spring.datasource.url=${dburl}
```

Comment:
Sorry i know i was the one who gave you these but could they change dburl to DBUrl and dbpassword to DBPassword